configuration/pih/pih-config-mexico-plan-alta.json

```diff
@@ -2,15 +2,14 @@
   "welcomeMessage": "EMR de Compañeros en Salud - Plan de la Libertad",
   "browserWarning": "Por favor utiliza <b>Google Chrome</b> como navagador",
 
-  "site": "PLAN_DE_LA_LIBERTAD",
+  "site": "PLAN_DE_LA_LIBERTAD_ALTA",
 
-  "primaryIdentifierPrefix": "PLA",
-  "providerIdentifierPrefix": "PLA",
+  "primaryIdentifierPrefix": "PLAA",
+  "providerIdentifierPrefix": "PLAA",
```

Comment:
No se si debemos cambiar esto.

@mogoodrich , are there consequences for changing the primary ID prefix of a server?

Reply:
For the primary identifier, it's not intended to be changed so it's not advisable... thinking it through, it *could* work, but we'd want to do some testing on a local/staging server first...  and it certainly wouldn't update the exisitng identifiers that had already been generated (though would be easy enough to write a liquibase changeset to do this).

@mseaton ?

Looking at the Provider Identifier, I think that can be changed, but, again, would be good to test.  (And we'd want to write a liquibase script to modify the existing identifiers, if that's what we want to do).

Reply:
@Joseph-Developer , debemos dejar los prefixos y `site` para el servidor de plan alta como tal. Ok?

Reply:
ok, ahorita hago el cambio

Reply:
el nombre del .json si esta bien que lo haya cambiado ?

Reply:
esta bien